articles/concepts/Glossary.md

```diff
@@ -12,31 +12,42 @@ uid: microsoft.quantum.glossary
+|<a id="partial-app"></a>Partial application|Calling a [function](xref:microsoft.quantum.glossary#function) or [operation](xref:microsoft.quantum.glossary#operation) without all the required inputs. This returns a new [callable](xref:microsoft.quantum.glossary#callable) that only needs the missing parameters (indicated by an underscore) to be supplied during a future application. For example, given the function `MyFunc(x : int, y : int) : int {return x + y;}` you can partially apply it to a new function `let NewFunc = MyFunc(_, 3)`. You can then call the new function at a later time with the missing parameter `NewFunc(2)` which returns the value *5*.  For more information, see [Partial application](xref:microsoft.quantum.language.expressions#partial-application).|
+|<a id="pauli"></a>Pauli operators|A set of three 2 x 2 unitary matrices known as the `X`, `Y` and `Z` quantum operations. The identity matrix, $I$, is often included in the set as well.  $I = \begin{bmatrix} 1 & 0 \\\\ 0 & 1 \end{bmatrix}$, $X = \begin{bmatrix} 0 & 1 \\\\ 1 & 0 \end{bmatrix}$, $Y = \begin{bmatrix} 0 & -i \\\\ i & 0 \end{bmatrix}$, $Z = \begin{bmatrix} 1 & 0 \\\\ 0 & -1 \end{bmatrix}$.   For more information, see [Single-qubit operations](xref:microsoft.quantum.concepts.qubit#single-qubit-operations).|
+|<a id="circuit"></a>Quantum circuit diagram|A method to graphically represent the sequence of [operations](xref:microsoft.quantum.glossary#operation) (or [gates](xref:microsoft.quantum.glossary#gate)) for simple quantum programs, for example ![Sample circuit diagram](~/media/qpe.png). For more information, see [Quantum circuits](xref:microsoft.quantum.concepts.circuits).|
+|<a id="quantum-libraries"></a>Quantum libraries|Collections of [operations](xref:microsoft.quantum.glossary#operation), [functions](xref:microsoft.quantum.glossary#function) and [user-defined types](xref:microsoft.quantum.glossary#user-def) for creating Q# programs. The [Standard library](xref:microsoft.quantum.libraries.standard.intro) is installed by default. Other libraries available are the [Chemistry library](xref:microsoft.quantum.chemistry.concepts.intro), the [Numerics library](xref:microsoft.quantum.numerics.intro) and the [Machine learning library](xref:microsoft.quantum.machine-learning.concepts.intro).|
+|<a id="quantum-state"></a>Quantum state|In quantum computing, the information about a [qubit](xref:microsoft.quantum.glossary#qubit) that can be physically [measured](xref:microsoft.quantum.glossary#measurement). The quantum simulator uses this information to simulate how a qubit responds to operations. For more information, see [The Qubit](xref:microsoft.quantum.concepts.qubit).|
```

Comment:
This might seem to imply that 1) "quantum states" specifically refer to just qubits, and 2) that a "quantum state" is fully defined by the immediately measurable data (if that were the case, I think either there would be no reason for Q# to exist in the first place, or we would already be in the year 2100---too late in the day to think about which).
Perhaps the following?
```suggestion
|<a id="quantum-state"></a>Quantum state|The precise state of a quantum system, from which [measurement](xref:microsoft.quantum.glossary#measurement) probabilities can be extracted, as well as how it will evolve subject to any operation. The quantum simulator uses this information to simulate how qubits respond to operations. For more information, see [The Qubit](xref:microsoft.quantum.concepts.qubit).|
```